appdev/nativeapp/index.rst

```diff
@@ -2,3 +2,13 @@
 
 Native applications
 ===================
+
+The recommended way to build a native application on Ubuntu Touch is to use the `Qt <https://doc.qt.io/>`_ framework ( current version 5.9.5 ).
```

Comment:
I think you want a double underscore for the Qt link.

Also it probably makes sense to mention that version is the current version on Ubuntu Touch.